koku/masu/database/aws_report_db_accessor.py

```diff
@@ -440,3 +440,36 @@ def check_for_matching_enabled_keys(self):
                 LOG.info(log_json(msg="no matching enabled keys for OCP on AWS", schema=self.schema))
                 return False
         return True
+
+    def populate_ec2_compute_summary_table_trino(self, source_uuid, start_date, bill_id, markup_value):
+        """
+        Populate the monthly aggregated summary table for EC2 compute line items via Trino.
+
+        Args:
+            source_uuid (str): The unique identifier for the data source.
+            start_date (datetime.date): The date representing the start of the billing period to populate.
+            bill_id (int): The billing entry ID associated with the data being populated.
+            markup_value (float): The markup value to apply to the costs, if any.
+
+        Returns
+            (None)
+        """
+
+        year = start_date.strftime("%Y")
+        month = start_date.strftime("%m")
+        table_name = self._table_map["ec2_compute_summary"]
+        msg = f"Inserting records into {table_name} for source {source_uuid} for {year}-{month}"
+        LOG.info(msg)
+
+        sql = pkgutil.get_data("masu.database", f"trino_sql/{table_name}.sql")
+        sql = sql.decode("utf-8")
+        sql_params = {
+            "schema": self.schema,
+            "source_uuid": source_uuid,
+            "year": year,
+            "month": month,
+            "markup": markup_value or 0,
+            "bill_id": bill_id,
+        }
```

Comment:
I dont know if this helps at all but all our trino queries already log the context for us, so that might be why David didnt do it here as well. The `_execute_trino_raw_sql_query` takes in the context and passes it to `_execute_trino_raw_sql_query_with_description` which we log `LOG.info(log_json(msg="executing trino sql", log_ref=log_ref, context=ctx))`